src/oAuth2.js

```diff
@@ -113,7 +113,8 @@ export class OAuth2 {
 
     return this.http.fetch(exchangeForTokenUrl, {
       method: 'post',
-      body: json(data),
+      headers: { 'Content-Type': 'application/json; charset=utf-8' },
+      body: JSON.stringify(data),
```

Comment:
No I got the original error I mentioned when using json(data).

Sent from my iPhone

> On Mar 17, 2016, at 9:09 AM, Matt Broadstone <notifications@github.com> wrote:
> 
> In src/oAuth2.js:
> 
> > @@ -113,7 +113,8 @@ export class OAuth2 {
> >  
> >      return this.http.fetch(exchangeForTokenUrl, {
> >        method: 'post',
> > -      body: json(data),
> > +      headers: { 'Content-Type': 'application/json; charset=utf-8' },
> > +      body: JSON.stringify(data),
> just out of curiosity, does body: json(data) still work? As in, is it only required that we add the headers here?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>